components/Page_Layout/Merchandise.js

```diff
@@ -0,0 +1,31 @@
+export default function Merchandise() {
```

Comment:
also, make sure that your commit messages describe the feature that you are adding. Kinda like making a roadmap of what you’ve done in the past :)